tests/test_stream_daq.py

```diff
@@ -4,19 +4,20 @@
 from miniscope_io.utils import hash_video, hash_file
 from .conftest import DATA_DIR, CONFIG_DIR
 
+
 @pytest.mark.parametrize(
-    'config,data,video_hash',
+    "config,data,video_hash",
     [
         (
-            'stream_daq_test_200px.yml',
-            'stream_daq_test_fpga_raw_input_200px.bin',
-            '40047689185bdbeb81829aea7c6e3070bcd4673976e5836a138c3e1b54d75099'
+            "stream_daq_test_200px.yml",
+            "stream_daq_test_fpga_raw_input_200px.bin",
+            "40047689185bdbeb81829aea7c6e3070bcd4673976e5836a138c3e1b54d75099",
         )
-    ]
+    ],
 )
-@pytest.mark.timeout(30)
+@pytest.mark.timeout(60)
```

Comment:
This is a bit unrelated, but any test could theoretically get stuck, so should we just have something global in `pyproject.toml` instead? It's very minor, but I was wondering because you changed this and the decorator below.
```
[tool.pytest.ini_options]
timeout = "60"
```

Reply:
that is not a bad idea to just have a global timeout - i mean yeah i would consider any test taking longer than 60 seconds a failure for sure.